pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -204,7 +210,6 @@ func NewController(client kubernetes.Interface, options Options) *Controller {
 	}
 
 	c.nodes = sharedInformers.Core().V1().Nodes().Informer()
-	registerHandlers(c.nodes, c.queue, "Nodes", c.onNodeEvent)
```

Comment:
why remove?

Reply:
Actually, we do not handle any node events. this is a cleanup and also a reduce of  cpu usage